addons/project/migrations/9.0.1.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,187 @@
+---Fields in module 'project'---
+project      / project.project          / date (date)                   : NEW
+project      / project.project          / date_start (date)             : NEW
+project      / project.project          / label_tasks (char)            : NEW
+# New fields ---> Nothing to do
+
+project      / project.project          / members (many2many)           : DEL relation: res.users
+# Done ---> Field removed, no more project team, so renamed column to None to allow recovery with a future OCA module
+
+project      / project.project          / message_follower_ids (many2many): not a function anymore
+project      / project.project          / message_follower_ids (many2many): relation is now 'mail.followers' ('res.partner')
+project      / project.project          / message_follower_ids (many2many): type is now 'one2many' ('many2many')
+# Concerns module Mail (has to be done in another migration script specifically for that module) ---> Nothing to do
+
+project      / project.project          / state (selection)             : selection_keys is now '['cancelled', 'close', 'draft', 'open', 'pending']' ('['cancelled', 'close', 'draft', 'open', 'pending', 'template']')
+# Done ---> 'template' is converted to 'draft'
+# TODO - does this behavior make sense on the functional level?
+
+project      / project.project          / user_id (many2one)            : NEW relation: res.users
+# TODO - does someone know where user_id comes from? (probably related to Mail module)
```

Comment:
user_id is defined in project.py around 168. Meanwhile, it has been removed from account.analytic.account, so it needs to be renamed in that module and then copied into the project_project table. If I am thinking clear at this moment, it seems that in addition to the existing *move_field_m2o* API method we could use a new "move_field_o2m" method, or you could write a short SQL query instead.